src/components/StarRating/StarRating.jsx

```diff
@@ -0,0 +1,54 @@
+import { Component } from 'react';
+import { icons } from '../../utils/icons';
+import * as S from './StarRating.styles';
+
+class StarRating extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      stars: [],
+      rating: 0,
+      hovered: 0
+    };
+
+    this.state.stars.push(...Array.from({ length: 5 }, (_, i) => i + 1));
```

Comment:
please move `...Array.from({ length: 5 }, (_, i) => i + 1)` to `stars` property definition, no need to use `.push()` for that

Reply:
Ok, thanks. Already fixed.